src/thing_effects.h

```diff
@@ -145,7 +145,7 @@ enum ThingEffectElements {
     TngEffElm_Blood1,
     TngEffElm_Blood2,
     TngEffElm_Blood3,
-    TngEffElm_Unknown05,
+    TngEffElm_UnusedSlot05,
```

Comment:
We do in fact use the effect elements, but the come from the effects.toml. The names are there too.

Reply:
done